librz/include/rz_analysis.h

```diff
@@ -653,7 +653,11 @@ typedef struct rz_analysis_var_access_t {
 typedef enum {
 	RZ_ANALYSIS_VAR_STORAGE_REG,
 	RZ_ANALYSIS_VAR_STORAGE_STACK,
-	RZ_ANALYSIS_VAR_STORAGE_CFA,
+	RZ_ANALYSIS_VAR_STORAGE_REG_OFFSET,
```

Comment:
@thestr4ng3r @wargio take a look at this change please, for the early feedback.

Reply:
Some cases, I should print more details about the Location.

![image](https://github.com/rizinorg/rizin/assets/24983369/ecb1eb94-397b-4780-9401-2ba7c1f47b47)
![image](https://github.com/rizinorg/rizin/assets/24983369/b297e954-ee47-4953-895e-cc404c22241a)

Reply:
I personally think it's a helpful change, lets see what others think.

Reply:
As said elsewhere, what do CFA and loclist mean for a user?